### DISCUSSION TITLE
Interface Type Question

### DISCUSSION THREAD

**Discussion by mahmoudkhalil2** [netbox-community/netbox#12413] [Q&A] [answered]
Hello, now i have an Ethernet switch and the switch interfaces connected to the other devices using 10GB DAC cables, so this mean the interface is not have SFP and the SFP is built in the DAC cables, so I can choose the DAC cables in the Cabling page but I need to know what will be the interface type for this ports, because it think it is not correct to choose the interface type 10GB SFP+ when using the DAC cables because the SW interface is not have SFP and the SFP is built in the DAC cables

So, what is the correct interface type for the Switch ports when connected using 10GB DAC cables.

![image](https://user-images.githubusercontent.com/121564738/235467478-2c525935-0c5c-42ff-bda2-f735cf0fde7f.png)

Thanks in advance.

**Comment by mtinberg** [comment, marked as answer]
The port on the switch is still an SFP+ socket though which is what the interface type represents and doesn't change depending on whether you plug in an LR, SR or DA module into it.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: mahmoudkhalil2 ***@***.***>
Sent: Monday, May 1, 2023 9:31 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Interface Type Question (Discussion #12413)


Hello, now i have an Ethernet switch and the switch interfaces connected to the other devices using 10GB DAC cables, so this mean the interface is not have SFP and the SFP is built in the DAC cables, so I can choose the DAC cables in the Cabling page but I need to know what will be the interface type for this ports, because it think it is not correct to choose the interface type 10GB SFP+ when using the DAC cables because the SW interface is not have SFP and the SFP is built in the DAC cables

So, what is the correct interface type for the Switch ports when connected using 10GB DAC cables.

[image]<https://user-images.githubusercontent.com/121564738/235467478-2c525935-0c5c-42ff-bda2-f735cf0fde7f.png>

Thanks in advance.

**Reply by candlerb** [reply]
In addition, you can set the cable type to DAC.

**Reply by mahmoudkhalil2** [reply]
Thanks Mark.

@candlerb I already set the cable to DAC in the cable page, my question are I need to select the interface type of the switch ports to be 10GB SFP+or keep it other selection.

**Reply by candlerb** [reply]
The interface is still an SFP+ port.

Netbox doesn't model the SFP+ module (the thing that plugs into the port), except in as much as you can add an Inventory Item and add it to an interface.

Therefore, even with a fibre SFP+ module inserted, the cable is still considered as terminating in the interface itself.
